### PR TITLE
Cookie Banner

### PR DESCRIPTION
**Pull request by danielljohnson**
@evandana I'm putting this up so you can take a look. Couple questions:

- We should get the message from their legal team (if they have one?). I just wrote a generic one, but usually there's a privacy policy to link so and I don't think we have one of those either...
- It's the most basic thing I could think of design wise, I stink at design, so if you want it to look nicer I might need some help.

### CODE REVIEW

**Comment by evandana** [comment]
Sorry for not putting this in earlier:
> “This application uses cookies to improve website performance. We use cookies to personalize content and to analyze our traffic. We do not share your information with any outside partners.”

*Interactive ‘I consent to cookie policy’ button

File: src/components/view/Contact/index.js [hunk 1]
```diff
@@ -63,6 +65,7 @@ class Contact extends Component {
                         )}
                     </div>
                 )}
+                <CookieBanner />
```
Comment (evandana) [inline src/components/view/Contact/index.js]: Should we put this up at the App level so it's available even if someone first visits with a deep linked URL?

File: src/components/view/common/CookieBanner/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    handleClick(e) {
+        e.preventDefault();
+        window.localStorage.setItem('cookieConsent', true);
+        this.setState({'cookieConsent': true});
+    }
+
+    render() {
+        const showBanner = !this.state.cookieConsent ? (
+            <div className="cookie-banner">
+                <p>This website uses cookies, which are necessary to its functioning. By closing this banner, clicking a link or continuing to browse otherwise, you agree to the use of cookies.</p>
```
Comment (evandana) [inline src/components/view/common/CookieBanner/index.js]: Small update requested here to match copy in other comment.

**Review by evandana — CHANGES_REQUESTED** [review]

File: src/components/view/Contact/index.js (hunk 1 above)
Comment (danielljohnson) [reply src/components/view/Contact/index.js]: yea I'll change.

**Comment by danielljohnson** [comment]
ok changes made.

**Comment by danielljohnson** [comment]
@evandana is there anything else to do on this or can we merge?

**Review by evandana — APPROVED** [review]

**Pull request merged**